src/App.module.scss

```diff
@@ -1,38 +1,41 @@
-.App {
-  text-align: center;
-}
+@import url('https://fonts.googleapis.com/css2?family=Orbitron&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Orbitron&family=Rajdhani&display=swap');
```

Comment:
Looks like we are duplicating some stuff here - can we just have 1 of these lines?